config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-periodic-dra.yaml

```diff
@@ -279,6 +279,7 @@ periodics:
             - --gcp-nodes=100
             - --provider=gce
             - --env=KUBE_FEATURE_GATES=DynamicResourceAllocation=true
+            - --env=KUBE_GCE_PRIVATE_CLUSTER=false
```

Comment:
this seems unrelated any may cause other issue IIRC, we may have to enable the pull-through image cache

Reply:
this is related, see the comment here: https://github.com/kubernetes/test-infra/pull/35206#issuecomment-3114562138

I saw there is a taint on the node 
```
    - effect: NoSchedule
      key: node-role.kubernetes.io/control-plane
    - effect: NoSchedule
      key: node.kubernetes.io/not-ready
    - effect: NoSchedule
      key: node.kubernetes.io/network-unavailable
      timeAdded: "2025-07-24T16:38:48Z"
    - effect: NoExecute
      key: node.kubernetes.io/not-ready
      timeAdded: "2025-07-24T16:40:11Z"
 ```
 
 https://storage.googleapis.com/kubernetes-ci-logs/logs/ci-kubernetes-e2e-gce-100-node-dra-with-workload/1948421418286845952/artifacts/nodes/kubectl_get_nodes.yaml
 
 without this, the prometheus pods will not schedule on the control plane node

Reply:
That's not what that env does.

https://github.com/kubernetes/kubernetes/blob/5be5fd022920e0aa77e29792fffbb5f3690547b3/cluster/gce/config-default.sh#L501-L503